tests/test_nexus_client.py

```diff
@@ -56,3 +57,32 @@ def test_split_component_path_errors(
         nexus_mock_client.split_component_path(component_path)
 
     assert x_error in str(e.value)
+
+
+@pytest.mark.parametrize(
+    'url,expected_base', [
+        ('http://localhost:8081', 'http://localhost:8081/'),
+        ('http://localhost:8081/', 'http://localhost:8081/'),
+        ('http://localhost:8081/nexus', 'http://localhost:8081/nexus/'),
+        ('http://localhost:8081/nexus/', 'http://localhost:8081/nexus/'),
+    ]
+)
+def test_nexus_context_path(url, expected_base, mocker):
+    """
+    Check that the nexus context (URL prefix) is taken into account
+    """
+    class MockResponse:
+
+        def __init__(self):
+            self.status_code = 200
+
+        def json(self):
+            return '{}'
+
+    mocker.patch('requests.request', return_value=MockResponse())
+
+    NexusClient(NexusConfig(url=url))
+    requests.request.assert_called_once_with(
+        auth=('admin', 'admin123'), method='get', stream=True,
```

Comment:
the default credentials are  in nexus_config.DEFAULTS: https://github.com/thiagofigueiro/nexus3-cli/blob/master/src/nexuscli/nexus_config.py#L9-L10
```suggestion
        auth=(nexus_config.DEFAULTS['username'], nexus_config.DEFAULTS['password']), method='get', stream=True,
```